zipkin-thrift/src/main/thrift/com/twitter/zipkin/zipkinQuery.thrift

```diff
@@ -78,9 +78,6 @@ struct TraceCombo {
   4: optional map<i64, i32> span_depths # not set if no spans in trace
 }
 
-/** This is obsolete. Use Order.NONE for compatibility. */
-enum Order { TIMESTAMP_DESC, TIMESTAMP_ASC, DURATION_ASC, DURATION_DESC, NONE }
```

Comment:
out of curiosity, does this break anything for people who used the old definition?